### PR TITLE
[CU-3jtu9bg] fix: report_content to be nullable

### PR DESCRIPTION
**Pull request by RonnyChiang**

### CODE REVIEW

**Comment by WendellLiu** [comment]
Task linked: [CU-3jtu9bg report attribute-content to be nullable](https://app.clickup.com/t/3jtu9bg)

File: prisma/migrations/20220922160343_testnullable_content/migration.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/*
+  Warnings:
+
+  - You are about to alter the column `deletedAt` on the `Photos` table. The data in that column could be lost. The data in that column will be cast from `DateTime(0)` to `DateTime`.
+  - You are about to alter the column `resolvedTime` on the `Reports` table. The data in that column could be lost. The data in that column will be cast from `DateTime(0)` to `DateTime`.
+
+*/
+-- AlterTable
+ALTER TABLE `Photos` MODIFY `deletedAt` DATETIME NULL;
```
Comment (RonnyChiang) [inline prisma/migrations/20220922160343_testnullable_content/migration.sql:9]: 前一版migration 20220829100912_soft_delete_photos 
```
ALTER TABLE `Photos` ADD COLUMN `deletedAt` DATETIME NULL,
    ADD COLUMN `isDeleted` INTEGER NOT NULL DEFAULT 0;

```

File: src/controllers/photo-controller.js [hunk 2]
```diff
@@ -54,11 +54,11 @@ const photoController = {
         userId: userId,
       };
 
-      const content = req.body.content;
+      const content = req.body.content ? req.body.content : null;
       const photoId = Number(req.params.id);
 
       const reportService = new ReportService({ logger: req.logger });
-      const report = await reportService.postReport(photoId, content, reporter);
+      const report = await reportService.postReport(photoId, reporter, content);
```
Comment (RonnyChiang) [inline src/controllers/photo-controller.js:61]: 更改parameter順序

File: src/models/report/constant.js [hunk 3]
```diff
@@ -3,5 +3,9 @@ const statusMap = {
   deleted: 'deleted',
   noIssue: 'no_issue',
 };
+const reporterTypeMap = {
+  USERID: 'USERID',
```
Comment (WendellLiu) [inline src/models/report/constant.js:7]: USER_ID

**Review by WendellLiu — COMMENTED** [review]
all good but one naming issue

File: src/models/report/constant.js (hunk 3 above)
Comment (RonnyChiang) [reply src/models/report/constant.js]: fixed

**Review by WendellLiu — APPROVED** [review]

**Pull request merged**